test/led.js

```diff
@@ -38,6 +38,26 @@ var instanceProperties = [{
   name: "interval"
 }];
 
+var rgbProtoProperties = [{
+  name: "on"
+}, {
+  name: "off"
+}, {
+  name: "color"
+}, {
+  name: "toggle"
+}, {
+  name: "strobe"
+}, {
+  name: "blink"
+}, {
+  name: "stop"
+}];
+
+var rgbInstanceProperties = [{
+  name: "interval"
+}];
```

Comment:
This property is tested on `Led`, so I tested it here as well, but I would expect this to be hidden inside the private state in both cases.